### DISCUSSION TITLE
VS Code red squiggly lines

### DISCUSSION THREAD

**Discussion by iamfadhlu** [Cyfrin/foundry-full-course-cu#2730] [Q&A]
Hello everyone, my vscode doesn't highlight undefined parameters or variables and all with the red squiggly lines, I'm using the github dark contrast theme
Need to fix this so it helps identifying undefined variables or just something off

**Comment by EngrPips** [comment]
I use the extension in the image below to edit and format my solidity files. Maybe it will help you.

<img width="1440" alt="Screenshot 2024-10-23 at 14 14 44" src="https://github.com/user-attachments/assets/270de47a-30f0-41f5-b681-fe98e9293c70">
